.github/workflows/advanced.yml

```diff
@@ -152,40 +157,69 @@ jobs:
 
       - name: Get workflow SHA
         id: workflow-sha
+        if: ${{ env.PACK_TOOLCHAIN == 'true' }}
```

Comment:
What about rolling these steps to one step to get both keys?

Reply:
All steps obtaining cache keys has been merged into a single script.